app/main.py

```diff
@@ -24,6 +24,8 @@ class Config:
         'DELETE_FILE_ON_TRASHCAN': 'false',
         'STATE_DIR': '.',
         'URL_PREFIX': '',
+        'PUBLIC_HOST_URL': 'download/',
+        'PUBLIC_HOST_AUDIO_URL': 'audio_download/',

```

Comment:
If the value is changed to another path within the container, it won't work, because the routes are hard-coded in this file on line 212 and 213. There needs to be some solution for this.

Reply:
Not sure what L212-213 are used for but with my setup I forego those paths completely since I serve the files somewhere else (in this case a separate caddyserver)

As far as I can tell those would only apply for the default host values, but I guess a user might want to change the path prefix from `download/` to something like `output/`, is it that what you mean?

Reply:
Yes. But thinking about it again, there isn't really a need to support that. So we'll leave it as it is.